homeassistant/components/freebox/router.py

```diff
@@ -31,6 +31,16 @@
 SCAN_INTERVAL = timedelta(seconds=30)
 
 
+async def get_api(hass: HomeAssistantType, host: str) -> Freepybox:
+    """Get the Freebox API."""
+    freebox_path = Path(hass.helpers.storage.Store(STORAGE_VERSION, STORAGE_KEY).path)
+    freebox_path.mkdir(exist_ok=True)
```

Comment:
You're doing I/O in the event loop. That's not allowed.

Reply:
Fixed